### ISSUE TITLE
Relationships (ie ID)

### ISSUE THREAD

**Issue by jamesc892** [APIs-guru/graphql-faker#39]
Is there anyway to mock relationships? We have a use case where two seperate queries have an ID field, which in our production implementation actually would be the same, and we do some transformative work to lookup by matching IDs. 

ie. 
```js
{ field: 'thing1', id: 22 }
{ link: '22', id: 22 }
``` 

Currently faker randomizes IDs, but there seems to be no way to specify if a relationship should exist so that we can develop against "sane" faked data. 

Hopefully that's clear enough, wondering if there is a solution for this kind of scenario.

**Comment by IvanGoncharov** [comment]
@jamesc892 It's very hard to generate "sane" data in a tool that doesn't know your application logic.
I can suggest two approaches:
* you can restrict possible IDs by adding `@examples(values: [22, 23])` to ID field
* you can write a minimal GraphQL server with object having only `id` fields and use Faker in extend mode to add the rest: https://github.com/APIs-guru/graphql-faker#tldr

**Comment by jamesc892** [comment]
Yeah I agree that sane data in its entirety would be out of scope.

For context, we are trying to have on the fly introspection of our schema, and dynamically render faked data to avoid the need for engineers to do much from a configuration sense. Thinking thru the manual approach of restricting IDs and writing a minimal server to help were both scenarios we thought of, however they became a little too manual. We were hoping there was a flag that we could generate on our introspected schema to not rely on much static definition as we have a pretty robust API. We hoped something like a decorator @relationship(alias) that takes an alias that would link  two ids together could be feasible. I think we've reached limitations of this and other tools we've tried.

We'll reconvene here, and possibly throw you a PR if we think thru an aliased relationship approach, if that sounds interesting to you.

**Comment by IvanGoncharov** [comment]
@jamesc892 If you can suggest some general mechanism that would be useful to other users then I have no problem merging your PR or implementing it myself.

At the moment it hard for me to understand your proposal so can you please describe your setup step by step:
1. You are using `--extend` mode or fake entire schema? 
2. Can you please write simplified IDL/SDL?
3. Please provide two queries (you mentioned in the initial comment) against faked API with conflicting IDs.
